media/base/decoder_buffer.h

```diff
@@ -73,6 +77,36 @@ class MEDIA_EXPORT DecoderBuffer
     DiscardPadding discard_padding;
   };
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+class Allocator {
```

Comment:
nit: indent.   Please feel free to keep this as is if it's formatted by pre-summit.

Reply:
Fixed.